### PR TITLE
Language (behavior) fix

### PR DESCRIPTION
**Pull request by rschwammborn**
1. First commit includes fixes for minor documentation language errors in the installation pages.
2. Second commit adjusts the way Mapbender's language setting is handled. Starting with Mapbender v3.3, according to https://symfony.com/doc/4.4/translation.html#fallback-translation-locales

### CODE REVIEW

File: de/customization/yaml.rst [hunk 1]
```diff
@@ -202,9 +204,8 @@ Eine Konfiguration könnte wie folgt aussehen:
 
 .. code-block:: yaml
 
-   # locale en, de, it, es, ru, nl, pt are available
-    fallback_locale:   en
-    locale:            en
+   # en, de, it, es, ru, nl, pt are available
+    fallbacks:         en
```
Comment (astroidex) [inline de/customization/yaml.rst:208]: is it like that? Or does mapbender take the fallback locale of the browser? Info for 3.3 documentation

File: de/customization/yaml.rst [hunk 2]
```diff
@@ -187,7 +187,9 @@ Eine Konfiguration könnte wie folgt aussehen:
 Spracheinstellung
 *****************
 
-Die Sprache (locale) der Mapbender Installation kann angepasst werden, jedoch nicht die einer einzelnen Anwendung.
+Mapbender wird automatisch an die eingestellte Browsersprache angepasst.
```
Comment (astroidex) [inline de/customization/yaml.rst:190]: Vielleicht eher so: Mapbender verwendet automatisch die ausgewählte Sprache der Browsereinstellungen.

File: de/customization/yaml.rst [hunk 3]
```diff
@@ -202,9 +204,8 @@ Eine Konfiguration könnte wie folgt aussehen:
...
 .. code-block:: yaml
 
-   # locale en, de, it, es, ru, nl, pt are available
-    fallback_locale:   en
-    locale:            en
+   # en, de, it, es, ru, nl, pt are available
+    fallbacks:         en
     secret:            ThisTokenIsNotSoSecretChangeIt
 
 Weitere Informationen unter http://doc.mapbender.org/en/book/translation.html
```
Comment (astroidex) [inline de/customization/yaml.rst:211]: Please change the link to https://doc.mapbender.org/en/architecture/translation.html (book/... is the old link

File: en/architecture/translation.rst [hunk 4]
```diff
@@ -75,12 +75,12 @@ This is how a translation file messages.de.yml for German translation could look
 How can you activate translation?
 *********************************
 
-Activate your default locale in the configuration file **app/config/parameters.yml**
+Mapbender is automatically adjusted to your browser's language.
+Moreover, you can set the following parameter to your preferred language in the configuration file **app/config/parameters.yml**. If a translation should be missed, the set language instead will be used.
```
Comment (astroidex) [inline en/architecture/translation.rst:79]: ...the defined langauge will be used.

See comment above. Check with dev the behaviour

File: de/customization/yaml.rst (hunk 1 above)
Comment (astroidex) [inline de/customization/yaml.rst:208]: Does it change to 

fallbacks: ['en']

File: en/installation/installation_configuration.rst [hunk 5]
```diff
@@ -123,44 +122,44 @@ The basic configuration is done inside the **app/config/parameters.yml** file.
...
+general operation and a development environment in which the application can
 be tested. This concept follows the `"environments" in the Symfony framework
 <http://symfony.com/doc/current/book/configuration.html>`_.
 
-The production-environment is called with the URL
-http://localhost/mapbender/app.php, the development-environment with the
+The production environment is called with the URL
+http://localhost/mapbender/app.php, the development environment with the
 URL http://localhost/mapbender/app_dev.php. The call with app_dev.php is
 and should only be available from localhost.
```
Comment (astroidex) [inline en/installation/installation_configuration.rst:136]: default is only localhost. But you could edit the file at web/app_dev.php and add more IPs for further access

**Review by astroidex — CHANGES_REQUESTED** [review]

**Comment by rschwammborn** [comment]
Hi @astroidex , I took your comments in https://github.com/mapbender/mapbender-documentation/pull/315/commits/f59f456fa2baaea4e1d56ccd631998b63a02155f into account , please check again if everything is good.

**Comment by rschwammborn** [comment]
Please note that the base branch of this PR should be the upcoming 3.3 documentation, since new features are documented. I therefore changed the PR into a draft. Branch switch will occur as soon as the new doc branch is available.

**Comment by astroidex** [comment]
looks good.

**Review by astroidex — APPROVED** [review]

**Pull request merged**